core/rio/api/pom.xml

```diff
@@ -33,6 +33,11 @@
 			<version>${jaxb.version}</version>
 		</dependency>
 
+		<dependency>
+			<groupId>com.github.jsonld-java</groupId>
+			<artifactId>jsonld-java</artifactId>
+		</dependency>
```

Comment:
I understand why you need this here and I'm ok with it, but it does pinpoint to me that in a future release we really should move the format-specific helpers out of the api module, and into their respective parser implementation modules.

Reply:
Indeed, probably a todo for 4.0 ...

Reply:
Created #1755